src/Recipe.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import style from "./recipe.module.css";
+import { Recipes, RecipeDetails }  from "./types"
+
+// make types.ts
+const Recipe = ({ label, calories, image, ingredients }: RecipeDetails) => {
```

Comment:
Here, Recipe returns a React "Function Component", which is denoted as so:
`const Recipe: React.FC<RecipeDetails>`
Note: I'm not sure how important it is to include the generic type there. Might be worth researching into..